### ISSUE TITLE
len_zero: also handle .len() >= 1 

### ISSUE THREAD

**Issue by matthiaskrgr** [rust-lang-nursery/rust-clippy#2554]
Since ````.len()```` should return unsigned values in most cases, I think we can treat ````.len() >= 1```` the same as ````.len() != 0```` or am I missing something?

````rust
if String::from("hello").len() >= 1 {}
````
does not trigger a warning currently.

**Comment by oli-obk** [comment]
good idea!

> return unsigned values in most cases,

I think our `!= 0` check whitelists types that guarantee `usize`

**Comment by NiekGr** [comment]
Can I try to work on this?

**Comment by oli-obk** [comment]
All yours!

**Comment by NiekGr** [comment]
@oli-obk thank you.

**Comment by NiekGr** [comment]
ls it oké if the commits are based on v0.0.195? When rebasing to v0.0.196, with correct nightly, gives me build errors.

**Comment by NiekGr** [comment]
I reinstalled rustup, rustc, cargo, all of it and it compiles fine now on my end.

**Comment by matthiaskrgr** [comment]
Thank you! :D
I'm closing the ticket, not sure why it was left open.
